testdata/deep_exit_test.go

```diff
@@ -11,8 +12,17 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
 
+// Package level example
+func Example() {
+	log.Fatal(errors.New("example"))
+}
+
+// Testable example
 func ExampleFoo() {
-	if err := Foo("Hello, playground"); err != nil {
-		log.Fatal(err)
-	}
+	log.Fatal(errors.New("example"))
```

Comment:
It looks like Example() and ExampleFoo are duplicates. What is the point to have both?

Reply:
I tried to cover with tests all these functions:

https://go.dev/blog/examples#example-function-names

<img width="891" alt="image" src="https://github.com/user-attachments/assets/bd93fb10-5d13-4d0c-aaf3-b01976bca34c">